pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -67,6 +67,7 @@ const (
 	RegistryEditorRoleName = "registry-editor"
 
 	TemplateServiceBrokerClientRoleName = "system:openshift:templateservicebroker-client"
+	ClusterServiceBrokerAdminRoleName   = "clusterservicebroker-admin"
```

Comment:
Looking at this now, I wonder if this should be called `system:openshift:clusterservicebroker-admin`

WDYT @enj

Reply:
Yeah that sounds more appropriate to me (`system:openshift:clusterservicebroker-client` if you want to be consistent with the TSB role).  Also, please move this constant to `pkg/oc/bootstrap/docker/openshift/rbac.go` as having it here gives the false impression of being part of the bootstrap policy (the TSB is for now, but ideally it would not be).